interface-elements-for-desktop/articles/map/rulers.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Rulers
+author: Eugenia Simonova
+---
+
+# Rulers
+You can use rulers to measure distances and areas on a map.
+
+| Distance Ruler | Area Ruler | 
+|:---:|:---:|
+| ![Map Distance Ruler](../../images/map-ruler-distance.png) | ![Map Area Ruler](../../images/map-ruler-area.png)| 
+
+## Measure Distances
+
+Click  the ![Distance ruler button](../../images/map-ruler-distance-icon.png) button to create a distance ruler. This ruler calculates the distance between two or more points on a map. The total distance is shown in a map callout. Note that measurements ignore changes in elevation.
```

Comment:
```suggestion
Click  the ![Distance ruler button](../../images/map-ruler-distance-icon.png) button to create a ruler. This ruler calculates the distance between two or more points on a map. The total distance is shown in a map callout. Note that measurements ignore changes in elevation.
```

Reply:
"This ruler calculates the distance between two or more points on a map" Do you use the ruler to measure the distance, or does the ruler calculate the distance itself?

Reply:
resolved in teams

[8:59 AM] Eugenia Simonova (DevExpress)
    Good morning, Dirk (smile),
about https://github.com/DevExpress/dotnet-eud/pull/114

1.

"This ruler calculates the distance between two or more points on a map" Do you use the ruler to measure the distance, or does the ruler calculate the distance itself?

You click a map to set points and a ruler shows the distance value between these points. So, I would say that the ruler calculates the distance.

The same with area rulers. 


2.


"Click an intermediate point on the ruler to see the distance to this point."

What is the difference between 'a point' and 'an intermediate point'?


I meant:  intermediate point =not the first point and not the last point. 


If you click the first or the last point, nothing happens.


(1 liked)